modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/utils/types.ts

```diff
@@ -241,9 +241,10 @@ export type TViews = {
 	label?: string;
 	name?: string;
 	schema?: ISchema;
+	setItemComponentProps?: ({item, props}: {item: any; props: any}) => any;
```

Comment:
Unfortunately, Clay does not expose precise types of these components. If it did, `props` here would be union of three types. I think `any` here is better than duplicating those types locally.